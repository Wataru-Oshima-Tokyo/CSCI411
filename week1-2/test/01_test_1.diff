1,24c1,2
< Number of nodes and number of edges: 
< Add 5 edges: 
< 
< Original
< 0 1 2 3 4 
< 2 
< 3 4 
< 4 
< 1 
<  Transposed 
< 0 1 2 3 4 
< 4 
< 1 
< 2 
< 2 3 
< 0 0
< 1 0
< 2 0
< 3 0
< 4 0
< sccs size: 1
< id: 0 size: 4
< In: 0
< Out 0
---
> Number of nodes and number of edges: 
> Add 5 edges: 
