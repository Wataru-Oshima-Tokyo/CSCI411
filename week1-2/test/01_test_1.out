Number of nodes and number of edges: 
Add 5 edges: 
|A| = 0, |B| = 0, |C| = 4